storage.py

```diff
@@ -15,8 +15,9 @@ def get(self, key):
     def remove(self):
         pass
 
-    def set(self):
-        pass
+    def set(self, key, value):
+        if key in self.data:
```

Comment:
The set method should rise an exception KeyError. There was a similar questions in telegram chat;
![Screenshot from 2020-10-04 22-09-19](https://user-images.githubusercontent.com/22825575/95024867-70a64680-068e-11eb-8d89-ba68bf31f9ba.png)